src/styles/Wrapper.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+export const Wrapper = styled.div`
```

Comment:
This name is too abstract to produce reliable results for a new developer. I think you should clarify what is meant to be wrapped. Perhaps this is meant to be a module? I do already have a `PageWrap` component inside of `Layout.js` but I take it this is something different.